ontology/yaml/resources/units/units.yaml

```diff
@@ -225,6 +227,12 @@ specificenthalpy:
 - joules_per_kilogram_dry_air
 - kilojoules_per_kilogram_dry_air
 - megajoules_per_kilogram_dry_air
+speed:
```

Comment:
`linearvelocity` already exists. does that not work for this?

Reply:
wind actually cannot have a velocity right so i guess speed should be the right descriptor